aladdin/charts/merger/templates/values.yaml

```diff
@@ -0,0 +1 @@
+{{- toYaml .Values -}}
```

Comment:
this is just a dummy helm chart, but it allows us to merge yaml values file in the _exact_ same way that helm does (because we're using helm!)